packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/Artifact.java

```diff
@@ -25,12 +33,26 @@
 public class Artifact {
 
 	/* The id of the artifact. */
+	@Id
+	@GeneratedValue(strategy = GenerationType.UUID)
 	@TSOptional
+	@Schema(accessMode = Schema.AccessMode.READ_ONLY)
 	private UUID id;
 
-	/* Timestamp of creation */
+	@CreationTimestamp
```

Comment:
I guess that since these are not entities, then we don't need the @...timestamp and @column annotations?